packages/loader/container-loader/src/container.ts

```diff
@@ -995,9 +995,9 @@ export class Container
 				: combineAppAndProtocolSummary(summaryTree, this.captureProtocolSummary());
 
 		// Whether the combined summary tree has been forced on by either the supportedFeatures flag by the service or the the loader option or the monitoring context
-		const enableSummarizeProtocolTree =
-			this.mc.config.getBoolean("Fluid.Container.summarizeProtocolTree2") ??
-			options.summarizeProtocolTree;
+		const enableSummarizeProtocolTree = this.mc.config.getBoolean(
```

Comment:
Update comment that still says it can be set via loader options.

Reply:
Resolved